belgif-rest-problem-it/belgif-rest-problem-openapi-validation-spring-boot-3-it/src/test/java/io/github/belgif/rest/problem/RestProblemOpenapiValidationSpringBoot3IT.java

```diff
@@ -0,0 +1,32 @@
+package io.github.belgif.rest.problem;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+
+import io.restassured.RestAssured;
+import io.restassured.specification.RequestSpecification;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class RestProblemOpenapiValidationSpringBoot3IT extends AbstractOpenApiValidationSpringBootIT {
+
+    @LocalServerPort
+    private int port;
+
+    @Override
+    protected RequestSpecification getSpec() {
+        return RestAssured.with().baseUri("http://localhost").port(port).basePath("/spring");
+    }
+
+    // Only seems to work in SB3. In SB2 the atlassian lib throws the same exception, but the missing path seems not to
+    // be handled by a controller and not catched by the ControllerAdvice.
```

Comment:
I think you can add this test to superclass and override it adding @Ignore to the SB2 subclass?